src/zerocopy.rs

```diff
@@ -75,6 +100,16 @@ impl<T> Receiver<T>
 where
     T: OwningMessage,
 {
+    /// Receives a non-owning message from this channel.
+    ///
+    /// If there is a queued owning messaged, that message is immediately
+    /// popped. Otherwise, this function waits for the sender to send
+    /// non-owning data, which may be used directly.
+    ///
+    /// In order to preserve the safety of non-owning messages, the sender
```

Comment:
I feel like it is worth clarifying the precise meaning of "safety" as it is used here, given the sentence following this one.

Reply:
I'll correct this to "memory safety".